src/components/Scene.tsx

```diff
@@ -86,12 +88,12 @@ export const Scene: React.FC<SceneProps> = ({
             });
         }
 
+        const previewTag = `<preview ${previewUrl ? `url="${previewUrl}` : ''} 
```

Comment:
I believe the string here lacks a `"`, it should be `url="${previewUrl}"`.

Reply:
It's been a long time I didn't worked on krpano, I don't think i'll support this PR for the moment because i'm lacking of time, sorry.